UnitystationLauncher/Views/MainWindow.xaml.cs

```diff
@@ -1,19 +1,112 @@
 using Avalonia;
 using Avalonia.Controls;
+using Avalonia.Controls.Primitives;
+using Avalonia.Controls.Shapes;
+using Avalonia.Input;
 using Avalonia.Markup.Xaml;
+using System.Runtime.InteropServices;
 
 namespace UnitystationLauncher.Views
 {
     public class MainWindow : Window
     {
+
+        private Button _minimiseButton; //CustomButton Window
+        private Button _closeButton; //CustomButton Window
+        private Button _maximizeeButton; //CustomButton Window
+        private Path _maximizeIcon; //Update Icon
+        private ToolTip _maximizeToolTip; //Update ToolTip 
+        private DockPanel _titleBar; //For move WINDOW and hiding if user don't use Windows.
+        private Border _contentcontrol;  //If user don't use Windows OS then Grid.Row update and Grid.RowSpan
+
         public MainWindow()
         {
             InitializeComponent();
+
+            _titleBar = this.FindControl<DockPanel>("TitleBar"); 
+            _contentcontrol = this.FindControl<Border>("ContentControl");
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows) == false)
```

Comment:
Reverse this if statement so the windows-path comes first and the else path handles the other OSs